share/eds/store.go

```diff
@@ -53,17 +54,19 @@ type Store struct {
 	gcInterval time.Duration
 	// lastGCResult is only stored on the store for testing purposes.
 	lastGCResult atomic.Pointer[dagstore.GCResult]
+	fs           afero.Fs
 }
 
 // NewStore creates a new EDS Store under the given basepath and datastore.
-func NewStore(basepath string, ds datastore.Batching) (*Store, error) {
-	err := setupPath(basepath)
+func NewStore(basepath string, ds datastore.Batching, aFs afero.Fs) (*Store, error) {
```

Comment:
We shouldn’t use the Fs defined in the external dependency. The issue was about usage of the recently introduced FS interface as linked in the case.

Reply:
Now using the FS interface